src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/cli/DailyReportsImporterRunner.kt

```diff
@@ -0,0 +1,24 @@
+package uk.gov.justice.digital.hmpps.pecs.jpc.cli
+
+import org.springframework.boot.ApplicationArguments
+import org.springframework.boot.ApplicationRunner
+import org.springframework.boot.autoconfigure.condition.ConditionalOnNotWebApplication
+import org.springframework.context.annotation.Profile
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.pecs.jpc.config.TimeSource
+import uk.gov.justice.digital.hmpps.pecs.jpc.service.reports.ImportReportsService
+
+@ConditionalOnNotWebApplication
```

Comment:
Essentially this will run on startup of the application when the given profile is used.